code/modules/antagonists/blood brothers/brother.dm

```diff
@@ -0,0 +1,71 @@
+/*
+	*Blood brother is a team orientated antagonist that is meant to complete traitor style objectives using just their team members and game knowledge.
+	*blood brother should never be without a team.
+	*/
+/datum/antagonist/brother
+	name = "Brother"
+	roundend_category = "Brother"
+	job_rank = ROLE_BROTHER
+	var/datum/team/brother_team/team //the owning team is stored here.
+/*
+	* This proc is called when a new brother team is created.
+	*/
+/datum/antagonist/brother/create_team(datum/team/brother_team/new_team)
+	if(!new_team)
+		return
+	if(!istype(new_team))
+		stack_trace("Wrong team type passed to [type] initialization")
+	team = new_team
+
+/datum/antagonist/brother/get_team()
+	return team
+
+/datum/antagonist/brother/on_gain()
+	SSticker.mode.brothers += owner
```

Comment:
```suggestion
	SSticker.mode.brothers |= owner
```

The difference is that `|=` won't add to the list if the element is already in it.